src/components/ContentSidebar/SidebarAccessStats.js

```diff
@@ -37,13 +37,13 @@ const SidebarVersions = ({
 
     return (
         <SidebarSection title={<FormattedMessage {...messages.sidebarAccessStats} />}>
-            <AccessStatsComponent
+            <AccessStats
                 commentCount={comment_count}
                 downloadCount={download_count}
                 previewCount={preview_count}
                 editCount={edit_count}
                 openAccessStatsModal={onAccessStatsClick}
-                maxEvents={MAX_EVENTS}
+                isBoxNote={file.extension === BOX_NOTE_EXTENSION}
```

Comment:
Did you confirm that we won't want to blacklist other file types in the future?

Reply:
Think I am going to leave it like this for now. In the future I can potentially just send the extension to the component, and have the actual access stats feature determine what to show/hide